### DISCUSSION TITLE
RaspiNOAA V2 Disk Image

### DISCUSSION THREAD

**Discussion by ve3elb** [jekhokie/raspberry-noaa-v2#435] [General]
Hello Everyone. I have compiled and setup two full working images of Raspberry NOAA V2. 
Raspberry Pi Buster
Raspberry Pi Q4OS 

Some bug fixes and mods has been done and the images are very stable. 

Images are available here:
https://qsl.net/ve3elb/RaspiNOAA/

**Comment by dom-robinson** [comment]
Awesome - are they bug fixes we need to feed back into the master code?

Thankyou!

**Reply by ve3elb** [reply]
One of the big ones is that when installing the meteor_mod file is not installed in the correct folder. 

Then I changed the RGB color setting to match natural Colors.
